packages/unmock-core/src/service/state/middleware.ts

```diff
@@ -17,6 +17,22 @@ export default (state?: UnmockServiceState): IStateInputGenerator => ({
     spreadStateFromService(schema, filterTopLevelDSL(state || {})),
 });
 
+export const textMW = (
```

Comment:
If this is a general middleware pattern (meaning if the function signature is supposed to be the same for all middleware), could we use that here?

Reply:
The return type needs to be the same (`IStateInputGenerator`), but the signature itself (wrt to input) can be different as needed.